README.md

```diff
@@ -29,6 +29,7 @@
 
 ## History of Changes
 
+2022-07-13: Unit tests added using React Testing Library.
 2022-07-12: Math Magician application completed using react routes.
 2022-07-07: Calculator components refactored using hooks.
 2022-07-06: Calculator libraries and events handling added to provide calculator functionalities.
```

Comment:
- Please, kindly remove this section as it is not needed in the readme following the microverse [readme template](https://github.com/microverseinc/readme-template).
- Please, kindly update the live demo link of the app as it not taking users to the online version. You may choose to update the link or remove it totally to improve the user experience and to make your readme more professional.

Reply:
- Please, kindly update the "contributing" header by adding its missing content. you can refer to the  [readme template](https://github.com/microverseinc/readme-template) and ensure the link to the issue page of the calculator project is added to it.